utils/sb2-config-gcc-toolchain

```diff
@@ -226,7 +226,7 @@ if [ $? != 0 ]; then
 	exit 1
 fi
 
-SBOX_CROSS_GCC_DIR=$(dirname $(which $GCC))
+SBOX_CROSS_GCC_DIR=$(readlink -f $(dirname $(which $GCC)))
 if [ $GCC -v > /dev/null 2>&1 != 0 ]; then
```

Comment:
Please also fix this line while you are at it, it doesn't work in this state.
It should be:
```sh
if ! $GCC -v > /dev/null ; then
```

Reply:
done